src/core_plugins/tagcloud/public/__tests__/tag_cloud_visualization.js

```diff
@@ -65,14 +65,17 @@ describe('TagCloudVisualizationTest', function () {
 
     it('simple draw', async function () {
       const tagcloudVisualization = new TagCloudVisualization(domNode, vis);
+
       await tagcloudVisualization.render(dummyTableGroup, {
         resize: false,
         params: true,
         aggs: true,
         data: true,
         uiState: false
       });
-      const mismatchedPixels = await imageComparator.compareDOMContents(domNode.innerHTML, 512, 512, basicdrawPng, THRESHOLD);
+
+      const svgNode = domNode.querySelector('svg');
```

Comment:
this was a bug in the test-code. This PR (https://github.com/elastic/kibana/pull/15779) changed the internal DOM, and this unit test did not get updated accordingly. It didn't fail because comparison was too permissive, clouds have a lot of whitespace.